src/components/shared/ui/molecules/toast/index.tsx

```
@@ -0,0 +1,85 @@
+  update: (
+    id: string,
+    content: React.ReactNode | string,
+    options?: ToastOptions,
+  ): void => {
+    if (!toastRef.update) {
+      console.error(
+        "Toast provider not initialized. Make sure you have wrapped your app with ToastProviderWithViewport.",
+      );
+      return;
+    }
+    return toastRef.update(id, content, options);
```

Biome (2.3.14)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)